plasmapy/formulary/frequencies.py

```diff
@@ -516,3 +519,112 @@ def upper_hybrid_frequency(B: u.T, n_e: u.m**-3) -> u.rad / u.s:
 
 wuh_ = upper_hybrid_frequency
 """Alias to `~plasmapy.formulary.frequencies.upper_hybrid_frequency`."""
+
+
+@validate_quantities(
+    validations_on_return={
+        "units": [u.rad / u.s, u.Hz],
+        "equivalencies": [(u.cy / u.s, u.Hz)],
+    }
+)
+@angular_freq_to_hz
+def Buchsbaum_frequency(
+    B: u.T,
+    n1: u.m**-3,
+    n2: u.m**-3,
+    ion1: ParticleLike,
+    ion2: ParticleLike,
+    Z1: Optional[float] = None,
+    Z2: Optional[float] = None
+) -> u.rad / u.s:
+    r"""
+    Return the Buchsbaum frequency for a two-ion-species plasma.
```

Comment:
My earlier point on this still stands - it's still a restatement of the function name + list of arguments. I'd still prefer
```suggestion
    Return the ion-ion hybrid frequency for a two-ion-species plasma.
```
but of course, between the two of us you're the domain expert, and if this you say this is good enough, guess it is. Just making sure the earlier point is acknowledged and not accidentally missed!

Reply:
I see this definition as functionally the same as 'gyrofrequency.'
I'd welcome someone else to have a look at the wording, but I think it's ok.